elections/steering/2023/election_desc.md

```diff
@@ -1,11 +1,11 @@
-# Vote for the {{YEAR}} Steering Committee
+# Vote for the 2023-2024 Steering Committee
```

Comment:
This seems confusing.. isn't this just the 2023 election?

Reply:
Hi, @cblecker - You are correct that it's the 2023 election, and probably it could be called the 2023 Steering Committee - although this "year" runs through part of 2023 and part of 2024.

I based this update on what I saw in [2022](https://github.com/kubernetes/community/blame/master/elections/steering/2022/election_desc.md#L1) and [2021](https://github.com/kubernetes/community/blame/master/elections/steering/2021/election_desc.md#L1) but if you think it should say "2023" instead of "2023-2024" I can break with the convention of the last two elections.

Reply:
It's probably just nit picky, but yeah, I think we should talk about this being the 2023 election as even the term that folks will be elected for is 2 years.. so you're just just electing for a 12 month period.